test/index.spec.ts

```diff
@@ -145,10 +146,19 @@ describe('Unit testing for typedoc-plugin-versions', function () {
 				vUtils.refreshMetadata(metadata, docsPath).dev,
 				'v0.10.1'
 			);
+			// eslint-disable-next-line @typescript-eslint/no-var-requires
+			const currentVersion = require(path.join(
+				rootDir,
+				'package.json'
+			)).version;
```

Comment:
This can be simplified to `process.env.npm_package_version`, just tested this locally and by throwing it in this function and it prints the following via `npm test`:
```console
     Error: the string "0.2.1" was thrown, throw an Error :)
```
With this simplification we can discard the `rootDir` stub.

```suggestion
			const currentVersion = process.env.npm_package_version;
```